apps/omg_watcher/lib/exit_processor/in_flight_exit_info.ex

```diff
@@ -57,7 +60,10 @@ defmodule OMG.Watcher.ExitProcessor.InFlightExitInfo do
   @type t :: %__MODULE__{
           tx: Transaction.Signed.t(),
           # use utxo_position really, for convenience and tooling, even if oindex is always zero here
+          # this piece of data reflects the state of the contract
```

Comment:
oh right, then we need better names here or we'll regret:

`contract_tx_pos` and `tx_seen_in_blocks_at` (+ comment about what `nil` means for both)

Reply:
done